engines/ic3base.cpp

```diff
@@ -164,6 +157,10 @@ void IC3Base::initialize()
   solver_->assert_formula(
       solver_->make_term(Implies, trans_label_, ts_.trans()));
 
+  // assume property in pre-state
+  Term prop = smart_not(bad_);
+  solver_->assert_formula(solver_->make_term(Implies, trans_label_, prop));
```

Comment:
isn't it already done in the other PR #193

Reply:
Yeah I thought about that. But wouldn't it break things if we turned off that option? Also, if someone runs it through the API the pass from #193 won't run automatically. I figured it wouldn't be too bad to assume it twice to avoid those issues.

Reply:
Moreover, I think we need to do the same in `reset-solver`

Reply:
Yeah that's here: https://github.com/upscale-project/pono/blob/af84199796cb4f492ce8a9642a3b661765811143/engines/ic3base.cpp#L952. I was thinking about doing it in a separate PR, but it would probably be good to put all the label assuming code in one function since it's done in two places.

Reply:
> Yeah I thought about that. But wouldn't it break things if we turned off that option? Also, if someone runs it through the API the pass from #193 won't run automatically. I figured it wouldn't be too bad to assume it twice to avoid those issues. What do you think?

that is a good point. Let's keep it the way it is now